1. Basic Web Development/index.html

```diff
@@ -23,63 +22,122 @@
         <div class="square8"></div>
         <div class="square9"></div>
     </div>
-    <div class="preferences hidden">
-        <div class="bar"></div>
-        <div class="content">
-            <h1>Select your preferences:</h1>
-            <div class="section">
-                <h2>Presets:</h2>
-                <form>
-                    <div class="grid-element">
-                        <label for="all">All</label>
-                        <input id="all" name="preset" type="radio" value="All" />
+    <div id="site-content" class="no-display">
+        <div class="begin-arrow">
+            <p>Begin Editing</p>
+            <p> |</p>
+            <p> |</p>
+            <p>\/</p>
+            <a class="to-begin no-decor" href="#create">Edit Pop-up</a>
+        </div>
+        <div class="editor no-display">
+            <div id="creation-page" class="">
+                <a class="to-edit no-decor" href="#edit">To edit ></a>
+                <h1>Design your preferences pop-up</h1>
+                <form id="creation-form">
+                    <h2>Cathegory</h2>
+                    <div>
```

Comment:
is there any other tag that would be more descriptive than using `div` here?

Reply:
Are you referring to `span`?